clr-k8s-examples/8-kata/runc-runtimeClass.yaml

```diff
@@ -0,0 +1,6 @@
+kind: RuntimeClass
```

Comment:
@egernst why was this needed. Does containerd require runc to be explicitly listed?